packages/portal-proto/src/features/cart/CartHeader.tsx

```diff
@@ -476,7 +479,7 @@ const CartHeader: React.FC<CartHeaderProps> = ({
               Remove From Cart
             </Button>
           </Menu.Target>
-          <Menu.Dropdown>
+          <Menu.Dropdown data-testid="dropdown-menu-options">
```

Comment:
do you wanna differentiate this from the dropdown above?

Reply:
Nope, only one of those dropdowns can appear at a time. So I label every dropdown with the same data-testid so one function can work for all of them.